site/docs/install-ubuntu.md

```diff
@@ -3,25 +3,29 @@ layout: documentation
 title: Installing Bazel on Ubuntu
 ---
 
-# <a name="ubuntu"></a>Install Bazel on Ubuntu
+# <a name="ubuntu"></a>Install Bazel on Linux
 
-Supported Ubuntu Linux platforms:
+Supported Linux platforms:
```

Comment:
What's the full list of supported Linux platforms? Should I list everything that has an installer? (see https://github.com/bazelbuild/bazel/releases)